src/Rector/FuncCall/FactoryFuncCallToStaticCallRector.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Laravel\Rector\FuncCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\ClassConstFetch;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Expr\StaticCall;
+use Rector\Core\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see https://laravel.com/docs/7.x/database-testing#creating-models
+ * @see https://laravel.com/docs/8.x/database-testing#instantiating-models
+ *
+ * @see \Rector\Laravel\Tests\Rector\FuncCall\FactoryFuncCallToStaticCallRector\FactoryFuncCallToStaticCallRectorTest
+ */
+final class FactoryFuncCallToStaticCallRector extends AbstractRector
+{
+    /**
+     * @var string
+     */
+    private const FACTORY = 'factory';
+
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition('Use the static factory method instead of global factory function.', [
+            new CodeSample(
+                <<<'CODE_SAMPLE'
+factory(User::class);
+CODE_SAMPLE
+
+                ,
+                <<<'CODE_SAMPLE'
+User::factory();
+CODE_SAMPLE
+            ),
+        ]);
+    }
+
+    /**
+     * @return array<class-string<Node>>
+     */
+    public function getNodeTypes(): array
+    {
+        return [FuncCall::class];
+    }
+
+    /**
+     * @param Node\Expr\FuncCall $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        if (! $this->isName($node->name, self::FACTORY)) {
+            return null;
+        }
+
+        if ($node->args === []) {
+            return null;
+        }
+
+        $firstArgValue = $node->args[0]->value;
```

Comment:
need to have `isset($node->args[0])` check to avoid index removed by other rules.